app/views/pages/embedded.html

```diff
@@ -0,0 +1,74 @@
+{% extends "layouts/base.html" %}
+
+{% block content %}
+{% raw %}
+<style>
+  .alert {
+    display: none; !important;
+  }
+</style>
+
+<div>
+  <div class="waiter text-center"
+    ng-if="(state.isPackageLoading || state.isStarting)">
+    <i class="fa fa-spinner fa-pulse fa-4x"></i>
+    <span>Loading...</span>
+  </div>
+  <div ng-if="!state.isStarting">
+    <div class="col-md-12 col-xs-12 charts">
+      <babbage
+              ng-if="!state.flag.renderingCharts && currentTab == 'treemap'"
```

Comment:
Here and elsewhere in this PR: why do we have code like `currentTab == 'treemap'`? These embed routes just output a single view, so there are no tabs, nor need for `currentTab` or hard coded strings like `treemap` for conditions

?